fluid/resnet50.py

```diff
@@ -209,15 +209,12 @@ def run_benchmark(model, args):
         label = np.array(map(lambda x: x[1], data)).astype('int64')
         label = label.reshape([-1, 1])
 
-    iter = 0
     im_num = 0
-    start_time = time.time()
     for pass_id in range(args.pass_num):
```

Comment:
https://github.com/ccmteorljh/benchmark/blob/6e9b36b2cd318f308b80c984617f79f6eccdce5e/fluid/resnet50.py#L170    Maybe moving L170 before L213 is better.

Reply:
Done